.travis/install.sh

```diff
@@ -9,6 +9,12 @@ if [ ! -z "$USE_SYSTEM_GCC_VERSION" ]; then
 	sudo apt install -y gcc-${USE_SYSTEM_GCC_VERSION} g++-${USE_SYSTEM_GCC_VERSION}
 fi
 
+if [ ! -z "$USE_PYPY3" ]; then
+	sudo add-apt-repository ppa:pypy/ppa -y
```

Comment:
this is controversial. We use python from conda, but suddenly we use pypy from the host.

Shouldn't this be in meta.yaml?

This will require conda-forge, but it's better that a host dep.

Reply:
As a temporary work around, I'm okay with this.

Reply:
I tried to install pypy as a conda build/host dependency with ``python 3.6`` and set ``python 3.7`` as a run dependency.
This approach resolves the conflict between python versions in conda environment, but as a ``nextpnr-xilinx`` binary was built using ``python 3.6`` it is trying to link with libboost libraries for ``python 3.6`` and can't find them in the run environment.

For now I reverted to installation of ``pypy`` in the host.